configs/components/pxp-agent.rb

```diff
@@ -41,16 +66,21 @@
   end
 
   pkg.build do
-    ["#{platform[:make]} -j$(shell expr $(shell #{platform[:num_cores]}) + 1)"]
+    ["#{make} -j$(shell expr $(shell #{platform[:num_cores]}) + 1)"]
   end
 
   pkg.install do
-    ["#{platform[:make]} -j$(shell expr $(shell #{platform[:num_cores]}) + 1) install"]
+    ["#{make} -j$(shell expr $(shell #{platform[:num_cores]}) + 1) install"]
   end
 
+  # Due to windows, when we run the configure, we need this variable to be set
+  # in accordance with windows pathing. However, when we create the directory
+  # below, we need it in accordance with cygwin pathing.
+  install_root = settings[:prefix] if platform.is_windows?
```

Comment:
That doesn't appear to be true: https://github.com/puppetlabs/puppet-agent/blob/feature/master/windows-vanagon-development/configs/projects/puppet-agent.rb#L10-L12